src/components/Schedule/Schedule.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { DayInfoMessage } from "../Schedule/DayInfoMessage";
```

Comment:
rename file to index.tsx, so when you're importing schedule you can just simply write ```import Schedule from './components/Schedule``` instead of ```import Schedule from './components/Schedule/Schedule```